memory_statistics/test/memory_statistics_config.json

```diff
@@ -1,7 +1,8 @@
 {
     "lib_name": "Memory Statistics Test",
     "src": [
-        "test.c"
+        "test.c",
+        {"file": "testdir/empty.c", "tag": "tagged"}
```

Comment:
Do the file entries represent relative path from root of repository?

Reply:
Yes. They are either relative paths from repository root or absolute paths.

Reply:
Okay, thanks for clarifying. One minor improvement suggestion is to make the example more expressive about the utility of the "tag" feature by having entries for `first-party.c` and `third-party.c` files along with the namesake tags

Reply:
The idea is that we will be able to have "core_mqtt.c (coreMQTT)" in the corMQTT Agent table, and so on. coreHTTP would use a "(http-parser)" tag. So there really isnt a namesake tag.

Reply:
My intention of having that was to make the example expressive to mention that both first-party and third-party libraries are intended targets of the "tag" feature. (It wasn't about the namesake aspect). Anyway, I'm fine with the example as-is, so will not block the PR.